homeassistant/components/sensor/arlo.py

```diff
@@ -124,13 +153,31 @@ def update(self):
         elif self._sensor_type == 'battery_level':
             try:
                 self._state = self._data.battery_level
-            except TypeError:
+            except (AttributeError, TypeError):
```

Comment:
Why would there be an `AttributeError`?

Reply:
Yeah I don't think the attribute error should be necessary